recipes/speclite/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "speclite" %}
+{% set version = "0.7" %}
+{% set sha256 = "2753137e2f3dc0b94d6c3f98c24d674c1a7145410d731cbeb13e0e7f1f7f5ba2" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  sha256: {{sha256}}
+  
+build:
+  number: 0
+  noarch: python
+
+requirements:
+
+  build:
```

Comment:
I'm not sure if all of these packages are actually `build` requirements.

Reply:
I can get rid of matplotlib, for the others, they are needed otherwise `pip` will try to download them.